bundle/Resources/translations/ezrepoforms_content_type.en.xlf

```diff
@@ -58,6 +58,10 @@
                 <source>content_type.edit_title</source>
                 <target>Edit &lt;%contentTypeName%&gt;</target>
             </trans-unit>
+            <trans-unit id="15">
+                <source>content_type.publish</source>
+                <target>OK</target>
```

Comment:
"Ok" might not be the most clear action for publishing Content Type, should we go for "Publish"? Any input on what button should say form @rolandbenedetti maybe?